docs/install.rst

```diff
@@ -208,6 +218,12 @@ This way of running the tests may not work if you do it in the ``astropy`` sourc
 distribution. See :ref:`sourcebuildtest` for how to run the tests from the
 source code directory, or :ref:`running-tests` for more details.
 
+{%else%}
+
+See the `latest documentation on how to test your installed version of astropy <https://docs.astropy.org/en/latest/install.html#testing-an-installed-astropy>`__.
```

Comment:
# Dev

![doc_02_dev](https://user-images.githubusercontent.com/2090236/108885439-cfd10d00-75d5-11eb-9fe6-fba92c8818a1.jpg)

# Stable

![doc_02_stable](https://user-images.githubusercontent.com/2090236/108885454-d6f81b00-75d5-11eb-8b3c-045c601e7826.jpg)

Reply:
We could also add an intersphinx target for developer docs.  I'll try that out.

Reply:
I see you opened issue #11366 . 👍  It would be very nice if you can get it to work. 😸